### ISSUE TITLE
Debrid Search no longer working with Real-Debrid

### ISSUE THREAD

**Issue by jameSHIFT4** [MrMonkey42/stremio-addon-debrid-search#45]
Ever since sometime in the afternoon of Sunday, December 8, 2024 Debrid Search no longer pulls up any items that are in my Debrid cloud.

Upon searching for movie sources, it’ll just say “Debrid Search addon is still loading…” and then nothing comes up.

I tried resetting my API key but nothing seems to work. My other apps that work with Real-Debrid (torrentio) still seem to work normally.

**Comment by recital-cpu** [comment]
Yes not working for me either

**Comment by wenchiee** [comment]
+1

**Comment by MrMonkey42** [comment]
Seems to be working fine for me. Is anyone still having this issue?

**Comment by otakotario** [comment]
Yes, I'm still having the issue. For some anime, like Attack on Titan, the debrid search option used to appear in the sources drop-down menu, but for others, it didn’t. However, for the past 2-3 days, the debrid search option isn’t showing up at all—there’s no option in the drop-down menu anymore.

**Comment by otakotario** [comment]
Any update @MrMonkey42 ?

**Comment by ghost** [comment]
Still having this same issue

**Comment by mohayoub** [comment]
same issue

**Comment by lalkris97** [comment]
Even after the latest update yesterday, still its not working.

**Comment by MrMonkey42** [comment]
Seems to be some timeout related issue when fetching data from RD API. Looking into this further

**Comment by FlowersPowerz** [comment]
It works intermittently for me, let's say 80% of the time it doesn't work and 20% does, it's no longer reliable currently

**Comment by mohayoub** [comment]
> Seems to be some timeout related issue when fetching data from RD API. Looking into this further

Thanks for your efforts!

**Comment by lalkris97** [comment]
Hi... Just an update, it seems its working now 90% of the time

**Comment by gusfon83** [comment]
Not working for me. Is it just here or it's a general issue?

**Comment by ghost** [comment]
Has works 100% of the time for a few weeks now. Seems like it’s back to normal.
